doc/plotting.rst

```diff
@@ -66,7 +66,7 @@ Additional Arguments
 
 Additional arguments are passed directly to the matplotlib function which
 does the work.
-For example, :py:meth:`xray.DataArray.plot_line` calls 
+For example, :py:meth:`xray.DataArray.plot.line` calls 
```

Comment:
Thinking to just link these to `xray.plotting.line` function. Right now they don't work.